README.md

```diff
@@ -5,6 +5,56 @@
 
 An Android app for recording blood pressure measurements.
 
+## Pre-requisites
+
+The application currently requires JDK 1.8 to build. If you already have JDK 1.8 installed, skip this step.
+
+**I am not sure if I have the JDK installed, or I am not sure if I have the right version installed**
```

Comment:
Can we add a title here which says `FAQs`?

Reply:
This looks like a question. Should we add a `?` to all of them?
Or we can just rename them to 
1. `How to check JDK version?`
2. `How to install JDK?`

Reply:
Hmm, this is not exactly an FAQ, but sort of a POV of a reader who might be in a state of uncertainty about the JDK installation. I'll reword this to match your suggestions.

Reply:
Done.